_pages/docs/tutorials/beginner-tutorial.mdx

```diff
@@ -0,0 +1,213 @@
I/m6PYTKjX9W5cfn9ueH91DE9SnxUXREU/jF8TUfX324RE4p5frXmt3VXTd1DMzHBdxTW/N/SgHaWG7O7kKhTL5O0iw71wDqdaruHH7YN1JcJXE91wTc+TptJlauYzGe/zSjGiwvLPe8Wk87Vz+/zr35kJaQlAAACDlJREFU7dPG+GfdBnDXE+YJrHdMo3b8JVsef11SvSKMdymFzVFvsqQWEmbYOqZdWjlWN4SfOJ0cSYQBq8L0eWev9AIoYv2tQ1HehEujuOXSjpg+vFyeyIaf+X44U4ueZPz4evIhwvDOVn4fFd3UCbKUU/GuzfodKTfW6LedJ5JP2R1V9b6yaSyqWH/i+LGbl4QS3cazTXNxn7P4xPMzcaPbxZ3YM4l3M2MjWzCXzw6ly7jmXCyvPz7LSY74Dy/4mTKlbvI2F/Rm8uKKFXU85YkdC0qt0y9ycfSnA0qOYZN2ObgpCf/JY34LMbawgRaFPWzyQj17Cim3cDSbW+Lqbx7HKj92q0LWanFq8x6D9C3ebkua3bjw3McEgVezc3lI5h5fe3e0XwYghsjMeWOJcBEjp2ozUVl40cmrRp2j/DVOf91znxmPsNziF1THL686w4rAzQ8b3win/CP/Rf1f/1r93/l7n31rWfxfjuUv/qVG1PuIw8vf24T67XM2bP/TJfNIKm/z1vAuE+nmmLeikzf+w4NeNzMBRr6PAEWWm1p51Pf3BhLQkUhbPNukZEqWzEZYNq7euAWu+YxNWMc44Fw4kC3l0Vlur3M+ox79qA+DrfIwe7KO3bIWcXIrR7eJ/3f4Nv4MWJdVzLuXxYB8+9rmMN5dOjCC2Vz72OMGb37zfBcHpYE8u/7m6cWj3XWVtvmDcH0xMofSn+/uaOk3ECae/v1b1lQ/9ilgD4oqrWuPP+6db2nL49Xr46nHdeTL73wjFSE6AVw/nIlJ1fUlBR9M2BuLdra/WuX0p6yxFs98WV/HuHmADGt+1NsAVjnA2bFcy/NaxzeC8nFE+Pg9gxyGcSd+/pXiPeds5ERRP66tV7Y8z+obB/3Do2uOm4Oz3zj/9s/F9de6IdPpvAwGgbn2W8y/autTwff5eSZEOvKw/p6+gSNdZ7EX5WS/3E6P7KMfPHWy04WfZv05/pbj9r2hqqS2cMruGR3u+dXG+y4o5yTr7nHv2n5xne3Ok/MB5+JW/8XZPvkbFw1lsVz9cO/4jVnqvvMDN7Q+32haXl/841wP0b38dZ3yWvvlVVJi+RP7N54vzne8iwnDfi6n7H+4na+hpjO/5uX5PEzHvbhjMGG8MX37ED8r6WxZ/HO1+M/vYQ/DmH3zP474s7nxvJzM53HiZl7Ij3P429Nt2MH91a3+xZ5G+57dd30E5wf4uOL4TPX+lItzHOdvdfPrYnNc39svPmCxR/WtWcDzuXE4R8djES/Wr2xau9gfT748xDVXZk1KZL926p/SGNf+2DcbwRrNuh8SvflItKkv5lzVdf6kZ34+Z1cq9HwZlH+AvvNk7YbUZzq0FDkr6ROcTCesfv48x8aaMXf+SzNZnxkf2dr+5m2f6HshdS+MRomeoFJxaeUxd2QrlimsqLwk5oZhg0jZRDFJmeGT+teWYCh5nsb32I50P3DWH9smHnbZzBJp/vzsZetghfEmzXT474SLzgZ4A9LeqYxPecFZ8wnTdNb2q+86Ha++6t9ywVsmEYM1BjTErt3C/trtjFTny1V1mvFZ4LtcPr2qpq82ORRPDjwmfqJCdSRy46qzAxwy3E55O5TFce7LP51+VpCNTe3fuDCX98cN3i7Wg8d58wU7kwwvUBHr4biTUDuTcCe4NJ5Sx6c2Myicj2JlwrHvzVBq3qdpYkl/E1za4YtbH/Hh4wMP04lDvCZw9xVxvNg2sR6V4m/yfzmV/mzrM7Y+QiZObTYuXOO3J9BstF3M0ejPHHZx3U0RbNxcanuXZBfYWaz4cffhJp7J3Wp0iqRfiWr996j+YsVDpqwHG3/HKzJ85aEn/fD9e+5W3HD4JjlcbBbZHRuHeLzjNB7G06t48REd8v5PoQj6NnJx+We6sd1mSXDt5ZQfcZDT+YxL6uz7FeIZN3758MeP+Bs7efqclKK7Wy9Q4opACa74Ncf99UfS/KRPvxd1Uqlscxh/3IKqPzGNY6r0Ms7HAW956xyu7jbSmjSumjBsUuo3VXNC6XiVZnITI/x5bHn2xs4o+olhcrD8aJZfAmyM5JG1N1y6fwaITr8C3fHFIg5Gx6cxRiZPVxpOGh2XF2N5R8cFxJ08750jU3sXjU42ZTQ8Z50l5EEv1s2Ll7v0aJdXtdiM++QPr9I99QWqROBbFTuPqYiwJ/XYN94zSOw4CUosERfx2dTyBdz8Nd5qLH+XDB5Sv9hh7VwjjqK8OA4//dGRw6fw6ac9LZxUp3t5eaLZZ74vDzXqEx6DpTmIVl50R5nddKf3Wz9MsnvAaCg5djwL9OVWezqRbw2yWPuX+8na+Z7On2a5v7GCJT+NuX1WxjBFdnY9Ronf34jpfnRe/TcefV1bjeVHF4mU7VXW9fxWP+vvMNZeVtRnNZxyNj71WzCQr8jr04udfeO3nL/G/zpGyVptSsqp8ydt1zS/ukzmd2Ps40z49UUJvoDl8cL+Y7x2E7EcH2Zjiz5/6n/ye/uDcXphFLe63cnQGb9zSDCCOTwjtYff7p+6NlPR7+u9L97l/tY2/+M3rMPBV9/GjfRSMBzy0WGHmp/ncy6Ky+5yckcy901ye7L/L0fX1yiiPw+O20vZKPROpt191l7/h/w7dvD+MzZ2xTvMz7785UgnkJ9yc2kCejC2v/A93WalXJiWU3w4Knzew0TtpU+BMg9Kskr9FJX4g9q5f7FzwkMf9PNbbOe8ztt6cdZYnxc354SEeZ7OfvvUYmh35vlyVejaucYIB4IYHncvFtcdwLfed98QTHqR9xNCsfLOP132zvmwcPbhF23XP66jrIT/BdFFKMohHfzwAAAAAElFTkSuQmCC" />
```

Comment:
Ahh bit of a wrinkle here, we don't want to add images in this Base64 style for two reasons.

- It ruins the readability of the mdx files
- Also make the tutorial a little slower to load because the text and image data have to come together, when we link to the image url the image will be fetched after the text arrives to the browser.

Could you follow the convention in https://github.com/KittyCAD/website/blob/main/_posts/code-first.md?plain=1#L53
there the path refers to images that are in the [public website](https://github.com/KittyCAD/website/tree/main/public) directory.

FYI anything that is in the public directory is available through the website i.e. this image of our lawd and saviour https://github.com/KittyCAD/website/blob/main/public/jordannoone.jpg
Can also be accessed here https://kittycad.io/jordannoone.jpg

which is why `<img src="/path/in/public/dir"  />` works.

Which means you'll need to make a seperate commit to the website repo. Sorry it's a bit of a pain.